simulator/src/main/java/com/hedera/block/simulator/BlockStreamSimulator.java

```diff
@@ -28,39 +29,45 @@
 import java.lang.System.Logger;
 import java.nio.file.Path;
 
-/** The BlockStreamSimulator class defines the simulator for the block stream. */
+/**
+ * The BlockStreamSimulator class defines the simulator for the block stream.
+ */
 public class BlockStreamSimulator {
     private static final Logger LOGGER = System.getLogger(BlockStreamSimulator.class.getName());
 
-    /** This constructor should not be instantiated. */
+    /**
+     * This constructor should not be instantiated.
+     */
```

Comment:
There was no reason to expand this.

Reply:
✅ I reversed the formatting on the docs everywhere.